tutorials/roofit/rf401_importttreethx.py

```diff
@@ -68,26 +68,10 @@ def makeTTree():
 
 # Create a binned dataset that imports contents of all ROOT.TH1 mapped by
 # index category c
-dh = ROOT.RooDataHist(
-    "dh",
-    "dh",
-    ROOT.RooArgList(x),
-    ROOT.RooFit.Import("SampleA", hh_1),
-    ROOT.RooFit.Import("SampleB", hh_2),
-    Import=("SampleC", hh_3),
-    Index=c,
-)
+dh = ROOT.RooDataHist("dh", "dh", ROOT.RooArgList(x), c, {"SampleA": hh_1, "SampleB": hh_2, "SampleC": hh_3})
```

Comment:
```suggestion
dh = ROOT.RooDataHist("dh", "dh", ROOT.RooArgList(x), Index=c, Import={"SampleA": hh_1, "SampleB": hh_2, "SampleC": hh_3})
```

I think it's better to use here the constructor with the command arguments. With the additional keywords, it's easier to understand what the code does.